packages/adapters/ethers5/src/client.ts

```diff
@@ -219,7 +219,10 @@ export class Ethers5Adapter {
   public construct(appKit: AppKit, options: AppKitOptionsWithCaipNetworks) {
     this.appKit = appKit
     this.options = options
-    this.caipNetworks = options.networks
+    this.caipNetworks = CaipNetworksUtil.extendCaipNetworks(options.networks, {
```

Comment:
We already extending the networks in Appkit client, we don't have to do it again in ethers/ethers5/solana clients. This is only specific for Wagmi adapter that because it's getting `networks` prop and we need to extend them in Wagmi adapter, but not in the other adapters, they already getting extending to `construct` function

Reply:
Ahh that makes sense. I've refactored the code 🙏